ci/build-gmt.sh

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env bash
+# Build and install GMT
+
+# To return a failure if any commands inside fail
+set -e
+
+cat > cmake/ConfigUser.cmake << 'EOF'
+set (CMAKE_INSTALL_PREFIX "$ENV{INSTALLDIR}")
+set (GMT_LIBDIR "$ENV{INSTALLDIR}/lib")
+set (DCW_ROOT "$ENV{COASTLINEDIR}")
+set (GSHHG_ROOT "$ENV{COASTLINEDIR}")
+
+set (CMAKE_BUILD_TYPE Debug)
```

Comment:
Duplicate CMAKE_BUILD_TYPE if "COVERAGE" == "true".

Reply:
:+1: removed the one inside the if block